src/tests/base_test.py

```diff
@@ -644,7 +644,8 @@ def set_iptables_rules_for_nodes(
             for node in given_nodes:
                 given_node_ips.append(node.ips[0])
             cluster.nodes.shutdown_given(given_nodes)
-
+            # Clear cached info about nodes after shutdown
+            cluster.nodes.drop_cache()
```

Comment:
Why we need to do the shutdown in the first place? Any chance you know?

Reply:
The iptables fixture from base_test.py run shutdown, no idea if we really need shutdown.
We have failures in CI in tests that use this fixture and i suspect that when the libvirt comes back we dont have the latest ip.

Reply:
So let's try to remove the shutdown if it serves no purpose
There's no reason to just do more actions and increase our chances of failure

Reply:
How do you recommend to fix all other cases when we stop/ reboot nodes during test lifetime ?

Reply:
No, lets not remove the shutdown, it was implemented long time ago, there was a reason - if i am not mistaken if for the rules to apply, proxy tests are highly depended on it

Reply:
Let me check this a bit
Either way Adrien's suggestion to embed the cache-drop in the shutdown function is a better way to construct this solution

Reply:
Ack , yes Adrien's change will cover our needs but please note that we still may have an issue if user will try to call to Node().shutdown() .  but it good enough ,if no objection i am going to drop_cahce for: shutdown and reboot

Reply:
Seems like a good compromise